src/components/ebay-carousel/test/test.browser.js

```diff
@@ -80,13 +84,17 @@ describe('given the carousel starts in the default state with items', () => {
         });
 
         it('then it applies a translation', () => {
-            const translation = mock.itemWidth + constants.margin;
-            expect(list.style.transform).to.equal(`translateX(-${translation}px)`);
+            const { offsetLeft } = list.children[1];
+            expect(list.style.transform).to.equal(`translate3d(-${offsetLeft}px, 0px, 0px)`);
         });
 
         it('then it calculates item visibility correctly', () => {
-            expect(widget.firstVisibleIndex).to.equal(1);
-            expect(widget.lastVisibleIndex).to.equal(3);
+            const { state: { items } } = widget;
+            const visibleIndexes = [];
+            items.forEach((item, i) => {
+                if (item.fullyVisible) visibleIndexes.push(i);
+            });
```

Comment:
Maybe bring this function to the top so it can be reused across the tests that need this.